vmm/src/memory_manager.rs

```diff
@@ -716,6 +767,43 @@ impl MemoryManager {
                 .unwrap();
         }
 
+        // Apply NUMA policy if needed.
+        if let Some(nodes) = &mut host_numa_nodes {
+            let addr = region.deref().as_ptr();
+            let len = region.deref().size() as u64;
+            let mode = MPOL_BIND;
+            let mut nodemask: Vec<u64> = Vec::new();
+            let maxnode = (nodes.len() + 1) as u64;
```

Comment:
Let's say I use `--host-numa-nodes 1:128`, `maxnode` will be 2, but you will end up with a bitmask that is 128 bits long (or an array of 2 `u64`). So I think `maxnode` should be the max of your `nodes` array.

Reply:
OMG yes I completely messed up with this one...

Reply:
I did it more simply, based on the last node value.